src/websocket.js

```diff
@@ -26,32 +25,32 @@ const depth = (payload, cb) => {
         bidDepth: bidDepth.map(b => zip(['price', 'quantity'], b)),
         askDepth: askDepth.map(a => zip(['price', 'quantity'], a)),
       })
-    })
+    }
 
     return w
   })
 
-  return () => cache.forEach(w => w.close())
+  return () => cache.forEach(w => w.close(1000, 'Close handle was called', {keepClosed: true}))
```

Comment:
Please put spacing around objects as the lint command should tell you

Reply:
The linter was okay with my code:

![image](https://user-images.githubusercontent.com/469989/36995859-8dd86786-20b5-11e8-9926-d8abf3bb667f.png)

But I will put spaces! 😃